primitives/src/circuit.rs

```diff
@@ -1,7 +1,9 @@
 use crate::xtx::LocalState;
+pub use crate::SpeedMode;
```

Comment:
```suggestion
use crate::SpeedMode;
```

People can get this from crate anyway